api/test/examples/empty_paged_result.json

```
@@ -0,0 +1,6 @@
+{
+  "result": [],
+  "paging_metadata": {
+    "next_cursor": null
+  }
+}
```

```suggestion
}

```

Missing empty line at the end of the file